css/main.css

```diff
@@ -0,0 +1,377 @@
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 94px 24px;
+}
+
+.contact-me h2 {
+  text-align: center;
+  color: #172b4d;
+  font-size: 32px;
+  line-height: 34px;
+  margin-bottom: 34px;
+}
+
+.text-input {
+  height: 48px;
+  width: 97.5%;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding: 15px 38px 15px 16px;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+}
+
+.text-area {
+  height: 170px;
+  background: #fbf8f7;
+}
+
+.btn-submit {
+  display: block;
+  margin: auto auto;
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  border-radius: 8px;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+}
+
+.btn-submit:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
```

Comment:
- **Your design:** 

![image](https://user-images.githubusercontent.com/43172164/197035422-a4afa4f7-946c-4fac-90d2-2e3b8ac41984.png)

- **Figma design template:**

![image](https://user-images.githubusercontent.com/43172164/197035825-28041eae-2529-471a-95fa-0732694b3fb7.png)

- [ ] Kindly pay attention to the details by sticking to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible